### DISCUSSION TITLE
Column Layout is Missing

### DISCUSSION THREAD

**Discussion by widiu7omo** [filamentphp/filament#4363] [Help] [answered]
Split Layout not defined in Filament Table

![Screen Shot 2022-10-05 at 08 22 16](https://user-images.githubusercontent.com/25941348/193954160-c9417751-1ef7-4b89-8ef9-967e00f20986.png)

Inside the vendor folder, the composer not downloading Layout folder
![Screen Shot 2022-10-05 at 08 22 41](https://user-images.githubusercontent.com/25941348/193954262-93aabe65-eeba-4235-bb73-49ddd3eea225.png)

![Screen Shot 2022-10-05 at 08 24 33](https://user-images.githubusercontent.com/25941348/193954344-1845c77a-b6c8-45d4-9757-f0b70d88d5f3.png)
I'm already on the latest version

Is anyone facing this issue? 
Please help. Thanks.

**Comment by danharrin** [comment]
Please run `composer update` again, it's just not possible that you're on the latest version. Check your Filament version using `php artisan about`

**Reply by widiu7omo** [reply]
@danharrin , Thanks for your feedback. I already tried what you said.

But the result is still no luck. 

Here
![Screen Shot 2022-10-06 at 06 59 14](https://user-images.githubusercontent.com/25941348/194179557-74556dcc-82d5-40dc-9b9c-1742296eadaa.png)

![Screen Shot 2022-10-06 at 07 05 16](https://user-images.githubusercontent.com/25941348/194179635-b924c7c3-b0f6-43aa-a910-da95c7f13a20.png)

**Comment by danharrin** [comment, marked as answer]
In composer.json swap all `v2.16` for `^2.16` and run `composer update`.

**Comment by widiu7omo** [comment]
Finally. 
Thank you @danharrin. It's working now.
